core/tests/protractor_utils/ProfilePage.js

```diff
@@ -101,6 +113,9 @@ var ProfilePage = function() {
   };
 
   this.expectToHaveCreatedExplorationStat = async function(expectedStat) {
+    await waitFor.visibilityOf(
+      createdExplorationStat,
+      'Create exploration state is taking too long to appear');
```

Comment:
The `createdExplorationStat` element holds a statistic -- the number of explorations a user has created. Could you update the error message accordingly?